app/javascript/dashboard/App.vue

```diff
@@ -72,10 +96,27 @@ export default {
 
       if (accountId) {
         await this.$store.dispatch('accounts/get');
-        const { locale } = this.getAccount(accountId);
+        const {
+          locale,
+          latest_chatwoot_version: latestChatwootVersion,
+        } = this.getAccount(accountId);
         this.setLocale(locale);
+        this.latestChatwootVersion = latestChatwootVersion;
       }
     },
+    checkUpdateDismissedOrNot(version) {
+      return !this.getDismissedUpdates.includes(version);
+    },
+    dismissUpdateBanner() {
+      const updatedDismissedItems = this.getDismissedUpdates;
+      updatedDismissedItems.push(this.latestChatwootVersion);
+      localStorage.setItem(
+        'dismissedUpdates',
+        JSON.stringify(updatedDismissedItems)
+      );
+      this.showUpdateBanner = false;
+      this.latestChatwootVersion = this.globalConfig.appVersion;
```

Comment:
I would suggest creating a LocalStorage library to save and extract JSON data. Something like this https://gist.github.com/mritzco/72ac6ae497097b1657546b061a13706a